index.js

```diff
@@ -52,73 +52,49 @@ const root =
   // clicked on (in terminals such as iTerm) to open the file in an editor.
   `${process.cwd()}/`
 
-expandThunks = (array) => array.map(
+const expandThunks = (array) => array.map(
   elem => typeof elem == 'function' ? elem() : elem
 )
 
-makeLazyLogger = (strictLogger) => {
+const makeLazyLogger = (strictLogger) => {
   return strictLogger.enabled ?
     // if logger enabled, call it with functions expanded to actual values
     (...args) => strictLogger.apply(strictLogger, expandThunks(args)) :
     // no-op
     () => {}
 }
 
-makeStrictLogger = (label, {logToStderr, isErrorLogger}) => {
-  const logger = debug(label)
-  if (!logToStderr && !process.env.DEBUG_LOG_TO_STDERR) {
-    // by default log to stdout
-    logger.log = console.log.bind(console);
+const parseArgs = (args) => args
+  .map(arg => stringify(
+    arg instanceof Error ? {
+      name: arg.name,
+      message: arg.message,
+      data: arg.data,
+      stack: arg.stack
+    } : arg)
+  ).join(' ')
+
+const makeStrictLogger = (severity, context) => {
+  const logger = (...args) => {
+    const message = parseArgs(args)
+    const payload = { severity, timestamp: Date.now(), context, message }
+    console.log(stringify(payload))
```

Comment:
I'd like to be able to toggle (with an env var) between logging as single line of JSON and pretty printing. This is really useful when running things locally.

Reply:
so now by adding `PRETTY=true` you will get json indented with 2 places